### ISSUE TITLE
matchms similarity: Add MS2DeepScore metric

### ISSUE THREAD

**Issue by hechth** [RECETOX/galaxytools#276]
Models could again be passed as a parameter for the tool so that the model can be trained offline and then passed to the matching module.

- [ ] Similarity metric
- [ ] model training

AFAIK the models can be read using the function [here](https://github.com/matchms/ms2deepscore/blob/main/ms2deepscore/models/load_model.py) as a keras model.

The model could therefore be trained offline in the first stage and then later we could implement the training step as a separate tool.
